### PR TITLE
kubeadm: Make tabs with information how to install 3rd-party CNI network solutions

### PR DESCRIPTION
**Pull request by luxas**
Implements the proposal here: #2389

/cc @lukemarsden @errordeveloper @chenopis @alexellis @jbeda @pipejakob @timothysc

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/4274)

### CODE REVIEW

**Comment by luxas** [comment]
I was unable to get Romana working, so I left it out for now
Code to add later if it starts working:

	{% capture romana %}

	The official Romana set-up guide is [here](https://github.com/romana/romana/tree/master/containerize#using-kubeadm)

	**Note:** Romana works on `amd64` only.

	```shell
	kubectl apply -f https://raw.githubusercontent.com/romana/romana/master/containerize/specs/romana-kubeadm.yml
	```
	{% endcapture %}

/cc @chrismarino

File: docs/setup/independent/create-cluster-kubeadm.md [hunk 1]
```diff
@@ -229,13 +206,68 @@ You can install a pod network add-on with the following command:
...
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/projectcalico/canal/master/k8s-install/1.6/rbac.yaml
+kubectl apply -f https://raw.githubusercontent.com/projectcalico/canal/master/k8s-install/1.6/canal.yaml
+```
+{% endcapture %}
+
+{% capture flannel %}
+
+**Note:** For flannel to work correctly, `--pod-network-cidr=10.244.0.0/16` has to be passed to `kubeadm init`.
```
Comment (chenopis) [inline docs/setup/independent/create-cluster-kubeadm.md]: Same as above; turn these into bullet points under **Note**.

File: docs/setup/independent/create-cluster-kubeadm.md [hunk 2]
```diff
@@ -106,17 +106,16 @@ kubeadm on, and run:
 kubeadm init
 ```
 
+**Note:** You need to choose a Pod Network Plugin in the next step. Depending on what
```
Comment (chenopis) [inline docs/setup/independent/create-cluster-kubeadm.md]: Similar to below, combine these notes by making these bullet points under a single **Note:** label.

File: docs/setup/independent/create-cluster-kubeadm.md [hunk 3]
```diff
@@ -420,7 +474,9 @@ kubeadm deb/rpm packages and binaries are built for amd64, arm (32-bit), arm64,
 following the [multi-platform
 proposal](https://github.com/kubernetes/kubernetes/blob/master/docs/proposals/multi-platform.md).
 
-Currently, only the pod networks [flannel](https://github.com/coreos/flannel) and [Weave Net](https://www.weave.works/docs/net/latest/kube-addon/) work on multiple architectures.
+Only some of the network providers offer solutions for all platforms, please consult the list of
```
Comment (chenopis) [inline docs/setup/independent/create-cluster-kubeadm.md]: break into 2 sentences: "Only some of the network providers offer solutions for all platforms. Please consult the list of network providers above..."

**Comment by chrismarino** [comment]
Thanks @luxas I'll have a look and see what the problem is.

**Comment by cgilmour** [comment]
Hi @luxas, re "unable to get Romana working": could you open an issue at https://github.com/romana/romana/issues/new with some details about what didn't work?
I've just retested this and everything installed and launched as expected.

File: docs/setup/independent/create-cluster-kubeadm.md [hunk 4]
```diff
@@ -229,13 +206,68 @@ You can install a pod network add-on with the following command:
...
+The official Weave Net set-up guide is [here](https://www.weave.works/docs/net/latest/kube-addon/)
+
+**Note:** Weave Net works on `amd64`, `arm` and `arm64` without any extra action required.
+
+```shell
+kubectl apply -f https://git.io/weave-kube-1.6
+```
+{% endcapture %}
+
+{% assign tab_names = "Default,Calico,Canal,Flannel,Weave Net" | split: ',' | compact %}
```
Comment (lukemarsden) [inline docs/setup/independent/create-cluster-kubeadm.md]: Maybe "Default" would be better as "Choose one..."

**Comment by luxas** [comment]
Thanks! I now have the reviews I need to continue.
I'll fix the nits and then this is good to go :+1: 

@chrismarino confirmed to me that the default Romana manifest doesn't work OOTB with v1.7, so I'll let you add Romana when it's okay

**Comment by chrismarino** [comment]
@luxas I think we've got this covered now. Will confirm and get you something ASAP. Would like the Romana tab to be avail in this PR.

**Comment by chrismarino** [comment]
@luxas Can you please add this Romana tab back (see below). The problem you saw was a mismatch of interface names. That's been fixed so this url and yaml should be fine now.

{% capture romana %}

The official Romana set-up guide is [here](https://github.com/romana/romana/tree/master/containerize#using-kubeadm)

**Note:** Romana works on `amd64` only.

```shell
kubectl apply -f https://raw.githubusercontent.com/romana/romana/master/containerize/specs/romana-kubeadm.yml
```
{% endcapture %}

**Comment by luxas** [comment]
@chenopis should be ready to go now, fixed the comments

@chrismarino looked better now, thanks, readded!

**Comment by luxas** [comment]
Tech review has already been made :tada:

**Comment by luxas** [comment]
ping @chenopis

**Comment by chenopis** [comment]
@luxas Sorry, have been preoccupied for the past couple of weeks. :(

**Pull request merged**